packages/terra-application-navigation/src/ApplicationNavigation.jsx

```diff
@@ -171,6 +172,7 @@ const ApplicationNavigation = ({
     };
   }
 
+  // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Why did this need to be added when the line below hasn't changed?

Reply:
I had added that because of function `focusMainContent` throwing lint warning:
`The 'focusMainContent' function makes the dependencies of useLayoutEffect Hook`.

I can update it in a later PR with `useCallback` hook.